hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/RatisUtil.java

```diff
@@ -174,4 +178,18 @@ private static void setRaftSnapshotProperties(
         conf.getRatisSnapshotThreshold());
   }
 
+  public static void checkRatisException(IOException e, String port,
+      String scmId) throws ServiceException {
+    if (SCMHAUtils.isNonRetriableException(e)) {
```

Comment:
Here on server, we check this for NonRetriableException. But I don't see where we throw it.